tests/Main.hs

```diff
@@ -365,15 +365,22 @@ sinkEntrySpec =
 loadEntrySpec :: SpecWith (Path Abs File)
 loadEntrySpec =
   context "when an entry is loaded" $
-    it "is there" $ \path -> property $ \m b s -> do
+    it "is there" $ \path -> property $ \m b s r -> do
       let vpath = deriveVacant path
       B.writeFile (toFilePath vpath) b
+      addUTCTime (-60) <$> liftIO getCurrentTime >>= setModificationTime vpath
```

Comment:
Good, but can we then set time to some constant value istead, maybe 0 will work?